src/components/molecules/ChatList/ChatList.tsx

```diff
@@ -84,20 +87,26 @@ const ChatList: React.FC<ChatListProps> = ({
   const hideDeleteModal = useCallback(() => setShowDeleteModal(false), []);
 
   const hasMessages = hasElements(messages);
+
   const messageSender = usersById?.[messageToDelete?.from ?? ""]?.partyName;
+
+  // Last (newest) message goes first
+  const sortedMessages = sortBy(messages, ["ts_utc"]).reverse();
```

Comment:
Probably want to useMemo this

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/921/commits/f69bd1a816115991a5ca4257830f370237e55697